eng/scripts/Language-Settings.ps1

```diff
@@ -5,6 +5,50 @@ $packagePattern = "*.zip"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/python-packages.csv"
 $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=container&comp=list&prefix=python%2F&delimiter=%2F"
 
+function Get-AllPackageInfoFromRepo
+{
+  $allPackageProps = @()
+  Push-Location $RepoRoot
+  $allSetupProps = $null
+  try{
+    pip install packaging==20.4 -q -I
+    $allSetupProps = (python -c "import sys; import glob; import os; sys.path.append(os.path.join('scripts', 'devops_tasks')); from common_tasks import get_package_properties; \
+    obj=[get_package_properties(os.path.dirname(p)) for p in glob.glob('sdk/*/*/setup.py', recursive=True) \
+    if os.path.basename(os.path.dirname(p)) != 'azure-mgmt' and os.path.basename(os.path.dirname(p)) != 'azure' and os.path.basename(os.path.dirname(p)) != 'azure-storage']; \
+    print(obj)")
+  }
+  catch
+  {
+    # This is soft error and failure is expected for python metapackages
+    LogError "Failed to get all package properties"
+  }
+  Pop-Location
+  $allMatches = $allSetupProps | Select-String '\(.*?\)' -AllMatches
+  foreach ($matchInfo in $allMatches.Matches)
+  {
+    $matchValue = $matchInfo.Value -Split " "
+    $packageName = $matchValue[0].Trim("(',")
```

Comment:
Why not split on `,`?

Reply:
Yh, could do that too, but will still need to trim